docker/puppetdb/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:16.04
+
+ARG vcs_ref
+ARG build_date
+ENV PUPPETDB_VERSION="5.2.2"
```

Comment:
Don't know if it's appropriate, but we can produce the actual project.clj version via `lein pprint :version`.

Reply:
that command doesn't work for me:

```
morgan@river.corp.puppetlabs.net % lein pprint :version
Tried to load org.apache.httpcomponents/httpcore version 4.1.2 but 4.4.4 was already loaded.
Tried to load org.apache.httpcomponents/httpclient version 4.1.2 but 4.5.3 was already loaded.
Tried to load org.apache.maven.wagon/wagon-http version 2.2 but 2.12 was already loaded.
Tried to load commons-codec version 1.4 but 1.9 was already loaded.
Tried to load org.codehaus.plexus/plexus-utils version 3.0 but 3.0.24 was already loaded.
Tried to load org.codehaus.plexus/plexus-component-annotations version 1.5.5 but 1.7.1 was already loaded.
Tried to load org.jsoup/jsoup version 1.6.1 but 1.7.2 was already loaded.
Tried to load org.tcrawley/dynapath version 0.2.3 but 0.2.5 was already loaded.
Tried to load org.codehaus.plexus/plexus-interpolation version 1.14 but 1.24 was already loaded.
Tried to load bultitude version 0.2.0 but 0.2.8 was already loaded.
Tried to load commons-io version 2.0.1 but 2.5 was already loaded.
Tried to load com.cemerick/pomegranate version 0.3.1 but 0.4.0-alpha1 was already loaded.
Tried to load org.apache.maven/maven-repository-metadata version 3.0.4 but 3.5.0 was already loaded.
Tried to load org.apache.maven.wagon/wagon-provider-api version 2.2 but 2.12 was already loaded.
Tried to load commons-logging version 1.1.1 but 1.2 was already loaded.
Tried to load org.apache.maven/maven-model-builder version 3.0.4 but 3.5.0 was already loaded.
Tried to load org.apache.maven/maven-model version 3.0.4 but 3.5.0 was already loaded.
'pprint' is not a task. See 'lein help'.

Did you mean this?
         parent
```

Working on tooling for version is out of scope for this initial PR, I have another ticket in progress for that, but we want to get this initial work in first.

Reply:
Oh, sorry, should have said `lein with-profile ci pprint :version` since that plugin's currently guarded by the ci profile.

Reply:
ah, yeah, I don't think this will quite do what we want, we have no standard way to get SNAPSHOT versions (an investigation point I have going forward is how to build containers with, say, latest nightlies or something), but for right now we want to build with the latest released version so I think `git describe` will be a more reliable way to get that info

Reply:
Ahh, ok -- wasn't even sure of the context, just thought I'd mention it in case it was useful.